modules/administration-guide/partials/proc_using-chectl-to-install-che-on-codeready-containers.adoc

```diff
@@ -15,18 +15,18 @@ This section describes how to install {prod-short} on CodeReady Containers (CRC)
 
 .Procedure
 
-. Initiate the CRC platform:
-
-.. Configure your host machine for CodeReady Containers:
+.. Remove any previous cluster:
 +
 ----
-$ crc setup
+$ crc delete
```

Comment:
crc setup first.
But as it's part of the crc documentation, we could rely on it and skip the `crc setup ; crc delete` steps in our docs.

Reply:
why crc setup first? 
if there is an instance already, first you should delete it

Reply:
See: 
https://console.redhat.com/openshift/create/local
https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/2.0/html-single/getting_started_guide/index#setting-up-codeready-containers_gsg

successful `crc setup` is a prerequisite to run other `crc` commands `crc start` or `crc delete`

Reply:
These steps are covered by the prerequisites: An installation of CodeReady Containers. We should remove them.

Reply:
If there is an instance already, it means you already ran `crc setup`.

Reply:
well, I do not think we should remove them. Currently, the procedure is very well defined. If you have any concerns please provide the suggestion.

Reply:
```suggestion
```

Reply:
`crc delete` makes perfect sense to run before `crc setup`, @themr0c. The `delete` subcommand **does not** depend on the existence of files created during the `setup` command. As long as the `crc` executable exists in `$PATH`, you can safely run `crc delete`. If the instance doesn't exist already, no worries -- if it does, it deletes it. I believe this is safe to include in a general procedure.